src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -8,16 +8,15 @@ import { scrollAndFocus } from '../../common/utils/helpers';
 import PhotoPreview from '../components/PhotoPreview';
 import CropperController from '../components/CropperController';
 
-const LARGE_SCREEN = 1201;
 const MIN_SIZE = 350;
 
-function isSmallScreen(width) {
-  return  width < LARGE_SCREEN;
-}
-
-function onReviewPage(pageTitle) {
-  return pageTitle === 'Photo review';
-}
+const layouts = {
+  choosePhoto: 1,
```

Comment:
It makes sense, given the sequential nature of this flow, that these should have numeric values, however is there any interest in using strings instead ('choose_photo') similar to our redux constants for consistency sake, if that applies here?

Reply:
Updated this.